Jbarget/oxymore
# adds background to all pages

### What changes have you made?
- added backgrounds to all pages 
- used a different background to differentiate the nav menu overlay 
- reduced the file size of `background.jpg` from a whopping 32 mb to 3.4 mb but this is still too heavy and is slowing down the site 

### Which issue(s) does this PR fix?

Fixes #151 

### Screenshots (if there are design changes)
<img width="1431" alt="Screenshot 2020-10-09 at 12 36 01" src="https://user-images.githubusercontent.com/53219789/95573630-072a8d00-0a2c-11eb-8f7a-6a85a85cfc68.png">

### How to test
- nav through the site and see that the background image appears on every page 
- suggestions on speeding up load times mucho welcome
